README.md

```diff
@@ -1 +1,51 @@
-# Review-past-project
\ No newline at end of file
+# Review-past-project
+
+
+## Built With
+- Html
+- CSS
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+html/css/github workflow knowledge
+
+### Setup
+Set up and configure git on your system.
+
+### Install
+```
+git clone https://github.com/Kalunge/portfolio-project.git
+
+```
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@kalunge](https://github.com/kalunge)
+- Twitter: [@kalunge](https://twitter.com/titus_muthomi)
+- LinkedIn: [kalunge](https://linkedin.com/in/titus_muthomi)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](https://github.com//MIT-LICENSE.txt) licensed.
```

Comment:
fix this broken link

Reply:
- You spotted yet another broken link. Great effort, @Kalunge. Just as I mentioned in the previous inline comment, There should be some level of politeness, and your comment should be suggestive.

Reply:
- You spotted yet another broken link. Great effort, @Kalunge. Just as I mentioned in the previous inline comment, There should be some level of politeness, and your comment should be suggestive.